src/pages/Introduce/Introduce.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import './Introduce.scss';
+import { Link, useNavigate } from "react-router-dom";
+
+
+const Introduce = () => {
+    const navigate = useNavigate();
+
+    const handleInput = (e) => {
+        console.log('input 태그 동작')
+        // console.log(e.target.value)
+    }
+
+    const handleButton = () => {
+        console.log('button 태그 동작')
+        navigate('/product');
+    }
+
+
+
+    return(
+        <div className = 'Introduce'>
```

Comment:
```suggestion
        <div className = 'introduce'>
```
가장 부모요소의 className과 컴포넌트의 이름을 동일하게 해주신 부분은 잘 하셨지만
className은 소문자로 시작해야 합니다!